src/runtime_src/doc/toc/fpga_device_ready.rst

```diff
@@ -0,0 +1,54 @@
+.. _fpga_device_ready.rst:
+
+
+FPGA device readiness within user VM
+************************************
+
+To make the FPGA in user VM ready to use, the mgmt side has to be ready first. Beside that, depending on how
+the FPGA is deployed, there are still other factors to be considered.
+
+With MSD/MPD or Plugins used
+============================
+
+In this case, :ref:`mailbox.main.rst` has to be opened from user, and  MPD service must be active.
```

Comment:
user --> user VM

MPD service must be active --> MPD service must be active in the guest VM.

Reply:
changed